packages/cli-kit/src/store.ts

```diff
@@ -100,19 +150,19 @@ export class CLIKitStore extends Conf<ConfSchema> {
     return this.get('themeStore')
   }
 
-  setTheme(store: string): void {
+  setTheme(theme: string): void {
```

Comment:
The method is poorly named, but it is setting a store, not a theme. So this variable rename doesn't really make sense.

Reply:
Oops, I just changed the parameter by looking at the method name. I've renamed the methods as well.